.travis.yml

```diff
@@ -74,6 +74,7 @@ deploy:
   on:
     all_branches: true
     condition: $TRAVIS_EVENT_TYPE != cron && ! $TRAVIS_BRANCH =~ ^greenkeeper/
+    tags: false # Don't push tags to gh-pages
```

Comment:
It says setting tags "causes the branch condition to be ignored", so I think that means you don't need the all_branches line

Reply:
From what I can tell, without `all_branches`, it uses the default for `branch` which is whichever primary branch is configured in GitHub (develop on most repos), which would cause this step to only trigger on one branch. So I _think_ it's necessary to keep it.